modules/claims_api/app/services/claims_api/disability_compensation/pdf_generation_service.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require 'claims_api/v2/disability_compensation_pdf_mapper'
+require 'pdf_generator_service/pdf_client'
+
+module ClaimsApi
+  module DisabilityCompensation
+    class PdfGenerationService < ClaimsApi::Service
+      EVSS_DOCUMENT_TYPE = 'L023'
+
+      def generate(claim_id, middle_initial) # rubocop:disable Metrics/MethodLength
+        log_service_progress(claim_id, 'pdf',
+                             "526EZ PDF generator started for claim #{claim_id}")
+
+        auto_claim = get_claim(claim_id)
+
+        # Reset for a rerun on this
+        set_pending_state_on_claim(auto_claim) unless auto_claim.status == pending_state_value
```

Comment:
I'm also curious, specific to :18, can't we clean this up a bit? and perhaps that's a tech debt ticket, but at this point in time, we can't have anything outside of pending, so this line is a bit... pointless.